bank_integration/bank_integration/api/hdfc_bank_api.py

```
@@ -36,98 +36,164 @@ def login(self):
-                EC.visibility_of_element_located((By.NAME, "fldOldPass")),
-                EC.visibility_of_element_located((By.NAME, "fldMobile")),
-                EC.visibility_of_element_located((By.NAME, "fldAnswer")),
-                EC.visibility_of_element_located((By.NAME, "common_menu1")),
-            ),
-            throw="ignore",
-        )
+                throw="ignore",
+            )
+            # NOTE: The 'fldOldPass' and 'fldAnswer' conditions are not hit in the
+            # current HDFC UI. We still include them here so future maintainers know
+            # there is existing logic to handle password‑expiry and security‑question
```

Ruff (0.15.4)

[warning] Comment contains ambiguous `‑` (NON-BREAKING HYPHEN). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)